components/home/partners/logos/Raisedev.tsx

```diff
@@ -1,5 +1,10 @@
 export const Raisedev = () => (
-  <svg viewBox="0 0 114 22" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <svg
+    viewBox="0 0 114 22"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    width="1000px"
```

Comment:
Safari

Reply:
How did you figure these dimensions out?  That's cool.

Reply:
Oh, a max-width is set in the CSS so I just tacked a number of so if we ever increase the css we're unlikely to run into an issue with clashing width/max-width.